src/components/Dashboard.jsx

```diff
@@ -73,4 +58,12 @@ class Dashboard extends Component {
   }
 }
 
-export default Dashboard;
+const mapStateToProps = (state) => ({
```

Comment:
Im wondering if we can avoid some boilerplate using some react-toolkit features here instead of standard react-redux way

Reply:
I'm not sure what you mean by react-toolkit features but if you talk about hooks like `useSelector` or `useDispatch` then these can only be used in the functional components. I plan to rewrite this component into a functional one so I will remove this part in the process 👍